src/frontend/pages/_app.tsx

```diff
@@ -2,7 +2,7 @@ import { ThemeProvider as EmotionThemeProvider } from "@emotion/react";
 import { StylesProvider, ThemeProvider } from "@material-ui/core/styles";
 import { AppProps } from "next/app";
 import Head from "next/head";
-import React, { useEffect } from "react";
+import { useEffect } from "react";
```

Comment:
It seems like React is probably required to render the main component in our app ...